synapse/util/caches/descriptors.py

```diff
@@ -38,6 +40,8 @@
 
 logger = logging.getLogger(__name__)
 
+CacheKey = Union[Tuple, Any]
```

Comment:
isn't `Union[Tuple, Any]` the same as `Any` ?

Reply:
Yes, but it better represents the fact that it is either a single argument or a tuple of arguments.